app/helpers/question_view_helper.rb

```diff
@@ -14,9 +14,13 @@ def legend(question, page_header, conditional)
   end
 
   def answer_option_synonyms(resource)
-    return unless resource.instance_of?(Scheme)
+    return unless resource.instance_of?(Scheme) || resource.is_a?(Hash)
```

Comment:
any reason you didn't go for instance_of? here?

Reply:
No reason, I don't think it makes a difference here, so just preference